daffodil-test/src/test/scala/org/apache/daffodil/section16/array_optional_elem/TestArrayOptionalElemNew.scala

```diff
@@ -18,40 +18,28 @@
 package org.apache.daffodil.section16.array_optional_elem
 
 import org.apache.daffodil.util._
-import org.apache.daffodil.tdml.DFDLTestSuite
+import org.apache.daffodil.tdml.Runner
 import org.junit.AfterClass
+import org.junit.Test
 
 object TestArrayOptionalElemNew {
```

Comment:
We don't really use the scala-new thing anymore. I think it's reasonable to put this in TestArrayOptionalElem.scala. These *New.scala files should probably be deleted with their content moved to the not New files at some point. Maybe delete this file while you're at it?